forch/simple_auth_state_machine.py

```diff
@@ -13,24 +13,29 @@ class AuthStateMachine():
     UNAUTH = "Unauthorized"
     REQUEST = "RADIUS Request"
     ACCEPT = "Authorized"
-    MAX_RADIUS_RETRIES = 5
+    MAX_RADIUS_BACKOFF = 5
     QUERY_STATE_TIMEOUT = 10
-    REJECT_STATE_TIMEOUT = 20
-    AUTH_STATE_TIMEOUT = 20
+    REJECT_STATE_TIMEOUT = 300
+    AUTH_STATE_TIMEOUT = 3600
 
-    def __init__(self, src_mac, port_id, radius_query_callback, auth_callback):
+    # pylint: disable=too-many-arguments
+    def __init__(self, src_mac, port_id, auth_config, radius_query_callback, auth_callback):
         self.src_mac = src_mac
         self.port_id = port_id
         self.auth_callback = auth_callback
         self.radius_query_callback = radius_query_callback
         self.current_state = None
         self._retry_backoff = 0
         self.current_timeout = 0
```

Comment:
Maybe this can be a private member (name starting with "_")?

Reply:
done